pkg/blockstorage/vmware/vmware.go

```diff
@@ -115,7 +143,7 @@ func (p *fcdProvider) SnapshotCreate(ctx context.Context, volume blockstorage.Vo
 }
 
 func (p *fcdProvider) SnapshotCreateWaitForCompletion(ctx context.Context, snapshot *blockstorage.Snapshot) error {
-	return errors.New("Not implemented")
+	return nil
```

Comment:
What's the reason for making this a noop?

Reply:
SnapshotCreate will block until the snapshot is ready. That's why I decided to make it a noop.